met-etl/migrations/versions/2c7d5ce6e117_analytics_db.py

```diff
@@ -0,0 +1,210 @@
+    sa.ForeignKeyConstraint(['user_id'], ['user_table.user_id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('response_id', 'survey_id', 'engagement_id', 'user_id', 'request_key', 'request_label')
+    )
+    op.create_table('response_type_textarea',
+    sa.Column('response_id', sa.Integer(), nullable=False),
+    sa.Column('survey_id', sa.Integer(), nullable=False),
+    sa.Column('engagement_id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('request_key', sa.String(length=100), nullable=True),
+    sa.Column('response_value', sa.String(length=5000), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('active_flag', sa.String(length=1), nullable=True),
+    sa.ForeignKeyConstraint(['engagement_id'], ['engagement.engagement_id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['survey_id'], ['survey.survey_id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['user_id'], ['user_table.user_id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('response_id', 'survey_id', 'engagement_id', 'user_id')
+    )
+    op.create_table('response_type_textfield',
+    sa.Column('response_id', sa.Integer(), nullable=False),
+    sa.Column('survey_id', sa.Integer(), nullable=False),
+    sa.Column('engagement_id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('request_key', sa.String(length=100), nullable=True),
+    sa.Column('response_value', sa.String(length=5000), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('active_flag', sa.String(length=1), nullable=True),
+    sa.ForeignKeyConstraint(['engagement_id'], ['engagement.engagement_id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['survey_id'], ['survey.survey_id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['user_id'], ['user_table.user_id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('response_id', 'survey_id', 'engagement_id', 'user_id')
+    )
+    op.create_table('user_feedback',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('survey_id', sa.Integer(), nullable=False),
+    sa.Column('comments', sa.String(length=5000), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('sentiment_analysis', sa.String(length=100), nullable=True),
+    sa.Column('label', sa.String(length=100), nullable=True),
+    sa.ForeignKeyConstraint(['survey_id'], ['survey.survey_id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('user_id', 'survey_id')
+    )
+    op.create_table('user_response_details',
+    sa.Column('response_id', sa.Integer(), nullable=False),
```

Comment:
we will stick to either singular or plural for all table names..I think we can do plulars for all tables..whats your opinion?I am good to go with the standard patterns..
user_feedback could be user_feedbacks

Reply:
I made all to be singular instead, since most of the table names will be singular.